tests/manage/z_cluster/test_scc.py

```diff
@@ -0,0 +1,265 @@
+import logging
```

Comment:
I think there is some problem while rebasing this PR. please check and fix it.

Reply:
Sorry for misunderstanding. Curren PR was in WIP. I am closing it. Separated my changes from side commits to https://github.com/red-hat-storage/ocs-ci/pull/7178